### ISSUE TITLE
Add blog functionality

### ISSUE THREAD

**Issue by markbrough** [humportal/humportal#4]
We would like to include simple blog functionality in the Humportal. This would allow us to provide more visibility of some of the content we have developed over the last few years. In general, the blog should be just a short excerpt, then providing a link out to the relevant website. This is because we would like to keep traffic going to the devinit.org site where most blogs are currently hosted.

We also want to have the ability to capture featured work, including the following:
* a visualisation on funding to the [COVID-19 response](https://covid19.humportal.org/activities), which was then taken forward in [a visualisation by UN OCHA](https://data.humdata.org/viz-iati-c19-dashboard/);
* a tool to [explore funding to local and national actors in Somalia](https://iati-3w.humportal.org/), combining IATI and UN OCHA 3W data;
* a simple site and set of scripts to make it easier to [discover organisation identifiers for government entities](https://gov-id-finder.codeforiati.org/) in 50+ countries.

We therefore want to capture the following information:
* title
* date
* brief excerpt
* screenshot (optional)
* link to the relevant site
* category: blog or feature

We need to create a page where we would show this information -- e.g. `/blog/`. We should also consider how we could integrate e.g. the latest information on the front page.

We will begin to pull together blog examples and will provide these separately. All blogs etc. should be stored as individual markdown files, to make things simple to update over time. We should use `@nuxt/content@^1` (NB: we currently need to [use v1](https://content.nuxtjs.org/v1/getting-started/introduction), as we will continue to use NuxtJS v2 until v3 is released in a stable version).

In summary:
* [x] Setup basic blog functionality using `@nuxt/content@^1`
* [x] Design simple page showing individual blogs
* [x] Add `/blog/` route listing blogs
* [x] Consider adding blog entries to the home page.

**Comment by markbrough** [comment]
* MB to add sample blog data here:

**Comment by markbrough** [comment]
--
title: Somalia Local Activity Explorer
date: 2021-05-28
image: https://iati-3w.humportal.org/_nuxt/img/where.fd8b0b4.png
url: https://iati-3w.humportal.org/
category: feature
--
This prototype explorer throws a spotlight on the work of local and national aid actors in Somalia, and their relationships with regional and international partners. It combines the monthly humanitarian Somalia 3W (Who? What? Where?) reports compiled by UN OCHA, and the Somalia-focused activities that organisations self-publish through the International Aid Transparency Initiative (IATI).

**Comment by markbrough** [comment]
```md
--
title: Improving the visibility of local and national actors in humanitarian aid data
date: 2021-07-26
image:
url: https://devinit.org/resources/improving-visibility-local-national-actors-humanitarian-aid-data/
category: blog
--

The limited visibility of local and national organisations in humanitarian open aid data adds to the challenges of meeting and tracking the Grand Bargain commitment to provide more direct support to local and national actors. In support of the Grand Bargain transparency workstream and with funding from the Netherlands’ Ministry of Foreign Affairs, Development Initiatives (DI) explored the technical options for increasing the visibility of local and national actors in International Aid Transparency Initiative (IATI) data. This report highlights the technical methodology we pursued through our prototype tool development, key findings and recommendations for next steps.
```
